README.md

```diff
@@ -30,292 +31,4 @@ var constructorio = new ConstructorIOClient({
-| --- | --- | :---: |
-| `num_results` | string | ✔ |
-| `customer_ids` | array | |
-
-#### Send search result click event
-```javascript
-constructorio.tracker.trackSearchResultClick('dogs', {
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `name` | string | ✔ |
-| `customer_id` | string | ✔ |
-| `result_id` | string | |
-
-#### Send conversion event
-```javascript
-constructorio.tracker.trackConversion('dogs', {
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `name` | string | ✔ |
-| `customer_id` | string | ✔ |
-| `variation_id` | string | ✔ |
-| `revenue` | string | ✔ |
-| `section` | string | |
-| `result_id` | string | |
-
-#### Send purchase event
-```javascript
-constructorio.tracker.trackPurchase({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `items` | array | ✔ |
-| `revenue` | string | ✔ |
-| `order_id` | string | |
-| `section` | string | |
-
-#### Send recommendation view event
-```javascript
-constructorio.tracker.trackRecommendationView({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `url` | string | ✔ |
-| `pod_id` | string | ✔ |
-| `num_results_viewed` | number | ✔ |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `result_id` | string | |
-| `section` | string | |
-
-#### Send recommendation click event
-```javascript
-constructorio.tracker.trackRecommendationClick({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `pod_id` | string | ✔ |
-| `strategy_id` | string | ✔ |
-| `item_id` | string | ✔ |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `result_id` | string | |
-| `section` | string | |
-| `variation_id` | string | |
-| `num_results_per_page` | integer | |
-| `result_position_on_page` | integer | |
-
-#### Send browse results load event
-```javascript
-constructorio.tracker.trackBrowseResultsLoaded({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `url` | string | ✔ |
-| `filter_name` | string | ✔ |
-| `filter_value` | string | ✔ |
-| `section` | string | |
-| `result_id` | string | |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `selected_filters` | string or array | |
-| `sort_by` | string | |
-| `sort_order` | string | |
-| `items` | array | |
-
-#### Send browse result click event
-```javascript
-constructorio.tracker.trackBrowseResultClick({
-    parameters
-});
-```
-
-| Parameter | Type | Required |
-| --- | --- | :---: |
-| `item_id` | string | ✔ |
-| `filter_name` | string | ✔ |
-| `filter_value` | string | ✔ |
-| `variation_id` | string | |
-| `section` | string | |
-| `result_id` | string | |
-| `result_count` | integer | |
-| `result_page` | integer | |
-| `result_position_on_page` | integer | |
-| `num_results_per_page` | integer | |
-| `selected_filters` | string or array | |
-
-#### Receive status of tracking requests
```

Comment:
Should we leave this in or put it somewhere in the JSDocs?

Reply:
It's there! https://constructor-io.github.io/constructorio-client-javascript/module-Tracker.html#~on

Reply:
Nice!